src/main/java/pro/sky/java/coursework2/service/impl/JavaQuestionService.java

```diff
@@ -0,0 +1,59 @@
+package pro.sky.java.coursework2.service.impl;
+
+import org.springframework.stereotype.Service;
+import pro.sky.java.coursework2.domain.Question;
+import pro.sky.java.coursework2.service.QuestionService;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+@Service
+public class JavaQuestionService implements QuestionService {
+    private Set<Question> questions = new HashSet<>();
+
+    @Override
+    public Question add(String question, String answer) {
+        Question questionToAdd = new Question(question, answer);
+        questions.add(questionToAdd);
+        return questionToAdd;
```

Comment:
дублирование кода, надо просто делегацию сделать
```suggestion
        return add(new Question(question, answer));
```